packages/blocks/src/store/test/selectors.js

```diff
@@ -5,6 +5,7 @@ import {
 	getChildBlockNames,
 	isMatchingSearchTerm,
 	getGroupingBlockName,
+
```

Comment:
I'm not convinced that the unit test brings a lot of value here. I should add one or revert this empty line.